Cargo.toml

```diff
@@ -16,9 +16,9 @@ vendored = ["native-tls/vendored"]
 [dependencies]
 bytes = "0.5"
 native-tls = "0.2"
-hyper = { version = "0.13", default-features = false, features = ["tcp"] }
-tokio = { version = "0.2" }
-tokio-tls = "0.3"
+hyper = { version = "0.14", default-features = false, features = ["full"] }
```

Comment:
What is the reason for changing this to `full`?

Reply:
With `hyper-0.14`, `client`, `server` are not available by default, if you chose, `client`, `server`, `http1` and `http2` are also required, in addition to that `tcp` is required by `client::HttpConnector` and `runtime` and `stream` are default. Essentially, implying you need to get everything. 

I also tried with following features to make it compile `["client", "server", "http1", "http2", "tcp"]`, but then `full` is easier.

Reply:
This will be fixed in 0.14.2 https://github.com/hyperium/hyper/pull/2377